src/main/scala/mesosphere/marathon/core/storage/store/impl/zk/ZkPersistenceStore.scala

```diff
@@ -36,7 +37,7 @@ case class ZkId(category: String, id: String, version: Option[OffsetDateTime]) {
 }
 
 object ZkId {
-  val DateFormat = DateTimeFormatter.ISO_OFFSET_DATE_TIME
+  val DateFormat = Timestamp.formatter
```

Comment:
(1) There is now only one usage of this formatter which leaves me thinking whether it shouldn't be named differently, or is actually still worth a `val`. Maybe a verbose comment and two vals?
```
val DateWriteFormat = Timestamp.formatter
val DateReadFormat = DateTimeFormatter.ISO_OFFSET_DATE_TIME
```
Should they be used globally, i.e. in `GroupRepositoryImpl` as well?

(2) Also, can you add a test that verifies `Timestamp.formatter` is compatible to `DateTimeFormatter.ISO_OFFSET_DATE_TIME`? Otherwise, how do we _know_ it is?